lib/src/useCountdown/useCountdown.test.ts

```diff
@@ -1,10 +1,32 @@
+// TODO: check deprecate warning.
+
 import { act, renderHook } from '@testing-library/react-hooks/native'
 
-import useCountdown from './useCountdown'
+import useCountdown, { DEPRECATED_WARN } from './useCountdown'
 
 jest.useFakeTimers()
 
-describe('useCountdown()', () => {
+describe('deprecated useCountdown()', () => {
+  // deprecating.
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const packageJsonVersion = require('../../package.json').version
+  if (packageJsonVersion.startsWith('3.')) {
+    console.error(
+      'Skipping useCountdown test for version 3.x, please remove this test.',
+    )
+    return
+  }
+  test('should show deprecate warning', () => {
+    console.warn = jest.fn()
+    renderHook(() => {
+      useCountdown({ seconds: 60, interval: 500, isIncrement: false })
+    })
+    expect(console.warn).not.toHaveBeenCalledWith('some random string')
+    // expect(console.warn.mock.calls[0][0]).toBe(DEPRECATED_WARN)
+    // HELP! I cant type this `.mock`
```

Comment:
I'd be appreciated if someone can tell me how to type the .mock because I didn't see answers by my search. Might ask later on Stack Overflow.